### PR TITLE
[T4A4][W11-B3]Cherlynn Cha 

### PR DESCRIPTION
**Pull request by Cherlynn**

### CODE REVIEW

**Comment by Cherc** [comment]
Ready for review

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -165,5 +166,14 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    public static String getPrintableString(Printable...printables){
+        String printableString = "";
+        for(Printable next: printables){
+            printableString = printableString + next.getPrintableString();
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/ui/TextUi.java]: Be aware that there are a few ways to concatenate strings in Java, such as `StringBuilder`, `String.join`, etc.

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.data.person;
+
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Printable.java]: You forgot to add a header comment.

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**